examples/nlp/text_normalization_as_tagging/train_ru.sh

```diff
@@ -0,0 +1,21 @@
+NEMO_PATH=/home/aleksandraa/nemo
+DATA_PATH=/home/aleksandraa/data/tn_data/ru_pipeline13/datasets
+DATASET="itn_sample500k_rest1500k_select_vocab"
+LANG=ru
+LANGUAGE_MODEL=DeepPavlov/rubert-base-cased
+
```

Comment:
this should be moved to the training script comments, remove any hardcoded paths

Reply:
done